src/crud.js

```diff
@@ -35,14 +60,66 @@ export async function create (doctype, attributes) {
 export async function find (doctype, id) {
   const config = await waitConfig()
 
-  let path = '/data/' + (config.isV1 ? '' : `${doctype}/`) + id
+  if (!id) {
+    throw new Error('Missing id parameter')
+  }
+
+  let path = createPath(config, doctype, id)
   let response = await doFetch(config, 'GET', path)
 
   if (config.isV1) Object.assign(response, {_rev: NOREV})
 
   return response
 }
 
+export async function update (doctype, doc, changes) {
+  const config = await waitConfig()
+
+  const {_id, _rev} = doc
+
+  if (!_id) {
+    throw new Error('Missing _id field in passed document')
+  }
+
+  if (!_rev) {
+    throw new Error('Missing _rev field in passed document')
+  }
+
+  if (_rev === NOREV) {
+    changes = Object.assign({ _id }, changes)
+  } else {
+    changes = Object.assign({ _id, _rev }, changes)
+  }
+
+  let path = createPath(config, doctype, _id)
+  let response = await doFetch(config, 'PUT', path, changes)
+
+  if (config.isV1) return await find(doctype, _id)
+
+  return response.data
+}
+
+export async function _delete (doctype, doc) {
```

Comment:
because babel (or one tool of the build chain) would not allow it, throwing a "Unexpected token".. 😞 . if you have a way to get around this, i'm interested.